ui_testing/pages/contacts_page.py

```diff
@@ -0,0 +1,18 @@
+from ui_testing.pages.base_pages import BasePages
+
+class Contacts(BasePages):
+    def __init__(self):
+        super().__init__()
+        self.contacts_url = "{}contacts".format(self.base_selenium.url)
+
+
+    def get_contacts_page(self):
+        self.base_selenium.get(url=self.contacts_url)
+        self.sleep_small()
+
+    def get_random_contact(self):
+        row = self.get_random_contact_row()
+        self.open_edit_page(row=row)
+
+    def get_random_contact_row(self):
```

Comment:
no need to create a method which calls another method

Reply:
I edited it